spec/features/officing/residence_spec.rb

```diff
@@ -121,12 +121,21 @@
   end
 
   scenario "Verify booth", :js do
+    allow(Date).to receive_messages(
+                          :current => Date.new(2018,1,1),
+                          :today => Date.new(2018,1,1))
+    allow(Time).to receive(:current).and_return Time.zone.parse("2018-01-01 12:00:00")
+
     booth = create(:poll_booth)
     poll = create(:poll)
 
     ba = create(:poll_booth_assignment, poll: poll, booth: booth)
-    oa = create(:poll_officer_assignment, officer: officer, booth_assignment: ba)
-    create(:poll_shift, officer: officer, booth: booth, date: Date.current)
+    oa = create(:poll_officer_assignment,
```

Comment:
I think it's a good practice to name the variables as descriptive as possible...

Reply:
I changed the names to a more descriptive ones, and also changed the hardcoded '2018-01-01' date to a Date.current, because it's mocked before.